composer.json

```diff
@@ -1,16 +1,20 @@
 {
     "name": "nextcloud/ocp",
-    "description": "Composer package containing Nextcloud's public API (classes, interfaces)",
+    "description": "Composer package containing Nextcloud's public OCP API and the unstable NCU API",
     "type": "library",
     "license": "AGPL-3.0-or-later",
     "authors": [
         {
             "name": "Christoph Wurst",
             "email": "christoph@winzerhof-wurst.at"
+        },
+        {
+            "name": "Joas Schilling",
+            "email": "coding@schilljs.com"
         }
     ],
     "require": {
-        "php": "~8.0 || ~8.1 || ~8.2 || ~8.3",
+        "php": "~8.1 || ~8.2 || ~8.3",
```

Comment:
Sneaky

Reply:
:see_no_evil: 
Should have made it explicit, yeah